quran/fields.py

```diff
@@ -0,0 +1,44 @@
+from django.core import exceptions
+from django.db import models
+
+
+class SegmentsField(models.TextField):
+    description = "A custom field to save list of timecodes in DB as a string"
+
+    def to_python(self, segments):
```

Comment:
If you look at `CharField` docs you will see that they always call it `value`:
```suggestion
    def to_python(self, value):
```